src/ks-app/src/polyfills.ts

```diff
@@ -29,7 +29,6 @@ import 'core-js/es6/set';
 
 /* Evergreen browsers require these. */
 import 'core-js/es6/reflect';
-import 'core-js/es7/reflect';
```

Comment:
Just a question about this, I don't think its a blocker. 
Why is this being removed? I don't see why es6 would be kept instead of es7. Doesn't the es7 polyfill have everything in es6 plus new things?

Reply:
The Angular update tool removed this automatically, because this is only used in JIT. I believe the devkit now includes it as necessary instead of it being at the app level.

Reply:
See https://github.com/angular/angular-cli/commit/30f8352ca2b3cccd72f5fcff8e09a543f9c70a8d